setup.py

```diff
@@ -1,27 +1,6 @@
 #!/usr/bin/env python
 
-from distutils.core import setup, Extension
```

Comment:
Of course, this PR solves the problem by completely removing the C++ stuff 

If you'd rather keep it, there are some alternative approaches I can investigate (like making it optional - or splintering it to a different repo) but they will take more time